x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -61,7 +63,11 @@ export class IngestManagerPlugin
     });
   }
 
-  public start(core: CoreStart) {}
+  public async start(core: CoreStart): Promise<IngestManagerStart> {
+    const { isInitialized } = await core.http.post('/api/ingest_manager/setup');
```

Comment:
I am curious on how the app is going to react if this call throw an error

Reply:
Totally. I will explicitly handle it and return false for now and we can talk more about other options (retry then, later, etc)